### PR TITLE
Reduce jqwik dependency to jqwik-api

### PR DESCRIPTION
**Pull request by schlosna**
## Before this PR
jqwik testing dependency was pulling in more than necessary

## After this PR
==COMMIT_MSG==
Reduce jqwik dependency to jqwik-api
==COMMIT_MSG==

### CODE REVIEW

File: versions.props [hunk 1]
```diff
@@ -13,6 +13,7 @@ com.squareup.okhttp3:okhttp = 4.0.0
 com.uber.nullaway:nullaway = 0.7.9
 io.undertow:undertow-core = 2.0.20.Final
 net.bytebuddy:byte-buddy = 1.10.8
+net.jqwik:* = 1.2.4
```
Comment (carterkozak) [inline versions.props:16]: Should we add this to our managed artifacts to get automatic upgrades?

Comment (schlosna) [reply versions.props:16]: yes, PR to update the robots submitted.

File: tritium-registry/build.gradle [hunk 2]
```diff
@@ -20,7 +20,7 @@ dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter-params'
     testImplementation 'org.mockito:mockito-core'
     testImplementation 'org.mockito:mockito-junit-jupiter'
-    testImplementation 'net.jqwik:jqwik'
+    testImplementation 'net.jqwik:jqwik-api'
```
Comment (schlosna) [inline tritium-registry/build.gradle:23]: we don't need the full library, so updated to just use api

**Comment by schlosna** [comment]
will rebase after #657 merges

**Review by carterkozak — APPROVED** [review]

File: versions.props (hunk 1 above)
Comment (schlosna) [reply versions.props:16]: this is now just sorting all the versions

**Pull request merged**